branch-foreman

```diff
@@ -21,6 +21,8 @@ sed -i "/nightly/i \ \ \ \ \ \ - '${KATELLO_VERSION}'" centos.org/jobs/katello-p
 
 echo "- '${FOREMAN_VERSION}'" >> theforeman.org/yaml/includes/foreman_versions.yaml.inc
```

Comment:
if we're not doing koji releases anymore, can't this go?

(also, the file is now foreman_versions_koji, right?)

Reply:
No, this is the file with **all** releases. It's also used for Debian and other stuff like `smart-proxy-X.Y-stable-test` & `test_X_Y_stable`.